draft-reddy-tls-slhdsa.md

```diff
@@ -102,6 +102,8 @@ Applications that use SLH-DSA need to be aware that the signature sizes of the a
 
 Despite offering trade-offs between size and performance, all SLH-DSA variants produce significantly larger signatures than traditional signature algorithms. While SLH-DSA increases the size of the TLS 1.3 handshake, its impact on connection performance is minimal in the context of large data transfers, especially over low-loss networks. For instancee, TLS-based protocols are increasingly used to secure long-lived interfaces in critical infrastructure, such as telecommunication networks. In particular, DTLS-in-SCTP has been mandated in 3GPP for interfaces such as N2 that use long-lived TLS connections. 
 
+In deployments that aim to minimize handshake size, SLH-DSA may still be adopted for signing X.509 certificates while avoiding its use in TLS handshake messages. This avoids performance concerns associated with large signatures or expensive verification. SLH-DSA is a good choice for improving PQ security of root certificates and intermediate certificates without affecting TLS handshake performance.
```

Comment:
Should we mention a method that allows us to elide parts of the certificate at times?  If we can't, this advice doesn't make a great deal of sense.